docs/CHANGELOG

```diff
@@ -0,0 +1,125 @@
+v. 8.0.0-beta
+~~~~~~~~~~~~~
+
+Backend Support for Public and Private Clouds
+=============================================
+
+Zenko supports multiple-cloud storage operations natively using the Amazon
+Simple Storage Service (S3) protocol. Once a site has been integrated to
+the S3 protocol, Zenko handles S3 API calls and can replicate stored data at
+the site level to the following supported private and public clouds: 
+
+-  Amazon S3-based public clouds:
+
+   -  Amazon Simple Storage Service
+   -  Digital Ocean Spaces
+   -  Wasabi
+
+-  Amazon-S3-based private clouds:
+
+   - Scality RING S3 Connector
+
+-  Other public clouds:
+
+   -  Google Cloud Storage
+   -  Microsoft Azure Storage
+
+Other cloud storage services are being validated.
+
+CRR Retry
+=========
+
+As with any large data transfers over public infrastructure,
+cross-region replication operations are imperfect and can suffer
+failure. In early releases, CRR operations did not fail gracefully--sometimes
+requiring repetition of an entire CRR operation. With v. 1.0, Zenko provides

```

Comment:
I'm not sure how version numbers will work, but just making sure here, instead of 1.0, would this be 8.0.0-beta?